README.md

```diff
@@ -99,11 +99,15 @@ For now, productionizing is a bit manual. The minimum steps are:
   2. Don't expose `amppkg` to the outside world; keep it on your internal
      network.
   3. Configure your TLS-serving frontend server to conditionally proxy to
-     `amppkg`, if any of:
-     1. The URL starts with `/amppkg/`.
-     2. The URL is AMP and the `AMP-Cache-Transform` request header is present.
-        (Exercise caution not to send non-AMP URLs to `amppkg`; its
-        [transforms](transformer/) may break non-AMP pages.)
+     `amppkg`:
+     1. If the URL starts with `/amppkg/`, forward the request unmodified.
+     2. If the URL is AMP and the `AMP-Cache-Transform` request header is
```

Comment:
Is saying a "URL is AMP" common terminology?

Reply:
Updated. It seems "AMP page" is accepted terminology [1], much like "ATM machine" or "PIN number".

[1] https://www.ampproject.org/docs/fundamentals/discovery